### DISCUSSION TITLE
Some details on how to use Plotter.key_press_event() and Plotter.left_button_down()?

### DISCUSSION THREAD

**Discussion by chen112p** [pyvista/pyvista#2579] [Q&A]
Hello, it seems these two methods can let me achieve what I want to do... Can someone give me an example of the obj and event_type that are shown in the documentation? Thanks!

Junting

**Comment by banesullivan** [comment]
> Can someone give me an example of the obj and event_type

`obj` and `event_type` aren't often needed by the callback. See https://github.com/pyvista/pyvista/issues/2890 for a change to make this more clear.

the `obj` is the VTK object that triggered the event, and event type is the `vtk.vtkCommand.EventIds` type of the event.

I suspect you don't really need these, though there are examples in PyVista where we used the triggering object or the event:

Object usage:

https://github.com/pyvista/pyvista/blob/edba67486abaa473eb377eda5b6da93fde612ab1/pyvista/plotting/widgets.py#L1907-L1910

Event usage:

https://github.com/pyvista/pyvista/blob/edba67486abaa473eb377eda5b6da93fde612ab1/pyvista/plotting/render_window_interactor.py#L574-L583
